draft-ietf-tls-dtls13.md

```diff
@@ -1544,13 +1594,106 @@ to messages which appear to be out-of-epoch with a canned ACK
 message; in this case, implementations SHOULD rate limit how
 often they send such ACKs.
 
+# Connection ID Updates
+
+If the client and server have negotiated the "connection_id" 
+extension {{I-D.ietf-tls-dtls-connection-id}}, either side 
+can send a new connection ID which it wishes the other side to use
+in a NewConnectionId message:
+
+~~~
+   enum {
+       cid_immediate(0), cid_spare(1), (255)
+   } ConnectionIdUsage;
+
+   struct {
+       opaque cid<0..2^8-1>;
+       ConnectionIdUsage usage;
+   } NewConnectionId;
+~~~
+
+cid
+: Indicates the CID which the sender wishes the peer to use.
+
+usage
+: Indicates whether the new CID should be used immediately or is a spare.
+If usage is set to "cid_immediate", then the new CID MUST be used immediately
+for all future records. If it is set to "cid_spare", then either CID MAY
+be used.
+{:br}
+
+If the client and server have negotiated the "connection_id" extension,
+either side can request a new CID using the RequestConnectionId message.
+
+~~~
+   struct {
+   } RequestConnectionId;
+~~~
+
+Endpoints SHOULD respond to RequestConnectionId by sending a NewConnectionId
+with usage "cid_spare" as soon as possible. Note that an endpoint MAY ignore
+requests, which it considers excessive (though they MUST be ACKed as usual).
```

Comment:
I'm not sure how excited I am about RequestConnectionId.... Do you think we really need it?

Reply:
You should say that you MUST NOT send either of these messages if oyu didn't negotiate CID.